Kitodo/src/main/webapp/WEB-INF/templates/includes/processFromTemplate/searchEdit.xhtml

```diff
@@ -21,17 +21,17 @@
     <h:outputScript name="js/opacImport.js" target="body"/>
     <p:panelGrid layout="grid" columns="2" cellpadding="10">
         <!-- choose from existing processes  -->
-        <p:row rendered="#{ProzesskopieForm.useTemplates}">
+        <p:row rendered="#{CreateProcessForm.usingTemplates}">
             <div>
                 <p:outputLabel for="processSelect" value="#{msgs.selectFromExistingProcesses}"/>
-                <p:selectOneMenu id="processSelect" value="#{ProzesskopieForm.processForChoice}"
+                <p:selectOneMenu id="processSelect" value="#{CreateProcessForm.searchTab.templateProcess}"
```

Comment:
I think "templateProcess" is a missleading title, because we do also have templates.
Maybe 
`modelProcess`
`blueprintProcess`
`originalProcess`

something like that?